hack/linter.sh

```diff
@@ -9,8 +9,8 @@ fi
 
 linterVersion="$(golangci-lint --version | awk '{print $4}')"
 
-if [[ ! "${linterVersion}" =~ ^1\.5[89] ]]; then
-	echo "Install GolangCI-Lint version 1.58 or 1.59"
+if [[ ! "${linterVersion}" =~ ^v1\.6[0-9] ]]; then
```

Comment:
This doesn't work:

```
> make lint
./hack/gofmt.sh
./hack/linter.sh
Install GolangCI-Lint version 1.6x.x
make: *** [lint] Error 1
 > golangci-lint --version
golangci-lint has version 1.61.0 built with go1.23.1 from a1d6c56 on 2024-09-09T14:33:19Z

```

Reply:
@msvticket are you using windows ? how did you install  golangci-lint?

Reply:
Mac. Installing with homebrew

Reply:
This is linux:
golangci-lint has version v1.64.8 built with go1.23.2 from (unknown, modified: ?, mod sum: "h1:y5TdeVidMtBGG32zgSC7ZXTFNHrsJkDnpO4ItB3Am+I=") on (unknown)
New version should work for both